spark_client/domain.py

```diff
@@ -248,7 +248,7 @@ def dir_package(self) -> str:
 
     @property
     def template_dir(self) -> str:
-        return f"{self.dir_package}/resources/templates"
+        return "tests/resources/templates"
```

Comment:
I believe it is best to change the dir_packge property with 

```
@property
    def dir_package(self) -> str:
        return os.path.dirname(__file__)
```

We should also not move the templates from the package as these are needed by the functions in the main package (basically with the LightKube class, we populate templates to create resources in K8s as part of the API functions exposed by the LightKube class)

Reply:
I tested this and all tests seems to pass (in my local machine) without a spark-client snap installed

Reply:
Yes, clearly -- my plan was alike.
The current code above was a purposeful hack to check if the pipeline would run with this.
(As I wasn't sure about the internal execution environment of Github CI pipelines.)

That's why I put the "PR" in WIP state. I'm just hacking around to get a hint of how Github respondes. Especially when I'm so lucky to do all of this when Github has been totally broken for 2 days already :grinning: